src/interfaces/opty/contracts/IVault.sol

```diff
@@ -3,6 +3,9 @@
 pragma solidity >=0.6.0 <= 0.9.0;
 pragma experimental ABIEncoderV2;
 
+//  libraries
+import { DataTypes } from "../../../libraries/types/DataTypes.sol";
```

Comment:
- [ ] remove vault from this package.

Reply:
- [ ] Have the DataTypes library per every protocol and not in general

Reply:
Done